datadog_checks_dev/tests/tooling/manifest_validator/input_constants.py

```diff
@@ -48,7 +48,7 @@
             "Category::Cloud",
             "Category::Log Collection",
             "Supported OS::Windows",
-            "Supported OS::Mac OS",
+            "Supported OS::macOS",
```

Comment:
```suggestion
            "Supported OS::Mac OS",
```

We'll take care of `datadog_checks_dev` because it will mess up with the changelog